Wrappers/Python/test/test_weightedL2NormSquared.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np 
```

Comment:
This whole file is not a `unittest` and should not be in this directory

Reply:
at least with that file name as it will be discovered during test time and fail.